charts/linkurious-enterprise/templates/deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       {{- if .Values.backup.litestream.enabled }}
       - name: init-litestream
         image: {{ .Values.backup.litestream.image }}
-        args: ['restore', '-if-db-not-exists', '-if-replica-exists', '-v', '/data/server/database.sqlite']
+        args: ['restore', '-if-db-not-exists', '-if-replica-exists', '/data/server/database.sqlite']
```

Comment:
see https://github.com/benbjohnson/litestream/pull/475/files#diff-141fdde90cba111ee365dcbb51f4be7f2871ba982d5c22a13e90a799898d0ef2 
and as I already deployed new chart on enterprise develop it's failing about this missing parameter: 
│ init-litestream 2024/06/21 13:50:15 ERROR failed to run error="flag provided but not defined: -v"                                                                                                        │

Reply:
and doc https://litestream.io/reference/restore/